anesthetic/samples.py

```diff
@@ -260,16 +261,12 @@ def plot_2d(self, axes, *args, **kwargs):
 
         local_kwargs = {pos: kwargs.pop('%s_kwargs' % pos, {})
                         for pos in default_types}
-        for pos in default_types:
-            local_kwargs[pos].update(kwargs)
-            if pos not in types:
-                types[pos] = None
```

Comment:
The `local_kwargs[pos].update` seems missing now, right?

Reply:
Ah yes, I think that was a slightly overzealous edit on my part. That should be restored